<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Properties</class>
 <widget class="QDialog" name="Properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rsrc/resources.qrc">
    <normaloff>:/icons/logo</normaloff>:/icons/logo</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QListWidget" name="listIcon">
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>94</width>
         <height>84</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pageEditor">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="indentWidthLabel">
             <property name="toolTip">
              <string>Taille de l'indentation en caractère
(peut utiliser des espaces ou des tabulations).</string>
             </property>
             <property name="text">
              <string>Taille de l'indentation</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="indentWidthSpinBox">
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="indentUseTabsLabel">
             <property name="toolTip">
              <string>Utiliser des tabulations plutôt que des espaces pour l'indentation.</string>
             </property>
             <property name="text">
              <string>L'indentation utilise des tabulations</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="indentUseTabsCheckBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="tabWidthLabel">
             <property name="toolTip">
              <string>Nombre de caractères que fait une tabulation.</string>
             </property>
             <property name="text">
              <string>Taille des tabulations</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="tabWidthSpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>8</number>
             </property>
             <property name="singleStep">
              <number>2</number>
             </property>
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="tabIndentsLabel">
             <property name="toolTip">
              <string>La touche Tabulation ajoute une indentation plutôt qu'une tabulation
(si les tailles d'indentation et de tabulation sont différentes).</string>
             </property>
             <property name="text">
              <string>Tabulation indente la ligne</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="tabIndentsCheckBox"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="backspaceUnindentsLabel">
             <property name="toolTip">
              <string>Enlève une indentation plutôt qu'un caractère.</string>
             </property>
             <property name="text">
              <string>Retour Arrière désindente la ligne</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="backspaceUnindentsCheckBox"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="autoIndentLabel">
             <property name="toolTip">
              <string>Quand on revient à la ligne, garder la même indentation.</string>
             </property>
             <property name="text">
              <string>Indentation automatique</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QCheckBox" name="autoIndentCheckBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageShortcuts">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="margin">
            <number>3</number>
           </property>
           <item>
            <widget class="QTreeWidget" name="shortcutsTree">
             <property name="rootIsDecorated">
              <bool>true</bool>
             </property>
             <property name="headerHidden">
              <bool>false</bool>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <attribute name="headerDefaultSectionSize">
              <number>300</number>
             </attribute>
             <column>
              <property name="text">
               <string>Commande</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">Raccourci</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QKeySequenceWidget" name="keySequenceWidget" native="true"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nouveau Raccourci : </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageMetadata">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Auteur</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPlainTextEdit" name="auteurPlainText"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Licence</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPlainTextEdit" name="licencePlainText"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="dateCheck">
           <property name="text">
            <string>Mise à jour automatique des meta-données
à la sauvegarde du fichier</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QKeySequenceWidget</class>
   <extends>QWidget</extends>
   <header>qkeysequencewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rsrc/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Properties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Properties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listIcon</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
